WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -124,11 +124,15 @@ fileprivate extension CardPresentPaymentEvent {
 #if DEBUG
 #Preview {
     NavigationStack {
+        let totalsViewModel = TotalsViewModel(orderService: POSOrderPreviewService(),
+                                                      cardPresentPaymentService: CardPresentPaymentPreviewService(),
+                                                      currencyFormatter: .init(currencySettings: .init()))
```

Comment:
Same as above, up until now the entry point seems to accept only abstractions either from the payment façade, or the POS interfaces, however now it would require to inject a specific POS VM.

Reply:
Addressed in commit 35370c72d7b0f5f967bfaa9304c256416372c12f.